docs/overview/mana.md

```diff
@@ -0,0 +1,48 @@
+# Mana
+Mana is behind the magic of Koinos. Every KOIN inherently contains Mana, which is used when using the Koinos blockchain. And just like in video games, your Mana rechardges over time letting you continue to use Koinos forever!
+
+This document will guide you through the basics of Mana on the Koinos blockchain. By the end of the guide you will understand how Mana makes Koinos different from other blockchains, how to use, and how to manage your Mana so you can use Koinos free forever.
+
+---
+## Blockchain Fees
+On most other blockchains, you must pay a fee to use the blockchain. Blockchain fees are analagous to tolls on roads. Oftentimes, tolls are imposed to help pay for the road and requires those that use the road to pay more for their increased use. It is a fair way to disribute responsibility for a shared resource, but no one enjoys paying a toll.
+
+Just like toll roads, Blockchains have costs associated with them. There are people running the blockchain software on computers and those computers aren't free. There are many ways individuals and organizations can profit off of participation in a blockchain network and fees are one of those ways.
+
+---
+## Why are fees a problem?
+There are a number of problems with fees. They create barriers to entry, can be unpredictable, and are error prone.
+
+The most obvious problem with fees is that, by definition, a fee is lost. You pay the fee and lose the value forever. This makes most blockchains "pay to play". From this respect, fee-less blockchains, like Koinos, were innevitable. We have seen this same pattern play out in the past with other technologies. Take texting, for example. When it first became available you had to pay for every text you sent. Nobody liked paying for text messages. Unlimited texting is now an expected feature of all cellular plans. In the same way, Koinos being fee-less is the future of blockchains.
+
+Other than fees not being user friendly, they are usually collected in the blockchain's native token. The native token is the "base" token of a network. On Koinos, this is KOIN. On Ethereum, it is ETH. Requiring someone to purchase a cryptocurrency before being able to use a blockchain creates a significant barrier to entry, resulting in many people being unwilling to try blockchain technology in the first place. While Mana is a property of KOIN, and requires holding KOIN to use Mana, somebody else can use their Mana to pay for you to use Koinos. Many dAPPs will pay their user's Mana so they do not even need to own KOIN before using Koinos.
```

Comment:
In blockchain basics, we define native token as "coin". It's an industry standard but we both know Koinos' coin is implemented just like a token. Let's decide how we want to refer to KOIN and be consistent.

Reply:
dApp